raiden/network/rpc/client.py

```diff
@@ -519,14 +445,15 @@ def __init__(self, host, port, privkey):
         self.protocol = JSONRPCProtocol()
         self.sender = privatekey_to_address(privkey)
 
+        self.nonce_last_update = 0
+        self.nonce_current_value = None
+        self.nonce_lock = Semaphore()
+        self.nonce_update_interval = nonce_update_interval
+        self.nonce_offset = nonce_offset
+
     def __repr__(self):
         return '<JSONRPCClient @%d>' % self.port
 
-    @property
-    def coinbase(self):
```

Comment:
Should this and `find_block` be removed or should they stay in the client. Not sure if the will be used/are useful though.

Reply:
Raiden signs all transactions locally, we don't need coinbase ATM.

Also, it's not hard to add these back if the need arises, right now it's just code bloat.

Reply:
Fine for me, can you add some docs to the `__init__` method, then it's ready for merging.